includes/data-stores/class-wcs-customer-store-cached-cpt.php

```diff
@@ -32,19 +31,38 @@ class WCS_Customer_Store_Cached_CPT extends WCS_Customer_Store_CPT implements WC
 	 */
 	const _CACHE_META_KEY = '_wcs_subscription_ids_cache';
 
+	/**
+	 * Gets the legacy protected variables for backwards compatibility.
+	 *
+	 * Throws a deprecated warning if accessing the now deprecated variables.
+	 *
+	 * @param string $name The variable name.
+	 * @return WCS_Post_Meta_Cache_Manager_Many_To_One|WCS_Object_Data_Cache_Manager_Many_To_One Depending on the HPOS environment.
+	 */
+	public function __get( $name ) {
+		if ( 'post_meta_cache_manager' === $name ) {
+			wcs_deprecated_argument( 'WCS_Customer_Store_Cached_CPT::post_meta_cache_manager', '5.2.0', 'WCS_Customer_Store_Cached_CPT::object_data_cache_manager' );
+			return $this->object_data_cache_manager;
+		}
+	}
+
```

Comment:
Because I changed the `post_meta_cache_manager` variable to `object_data_cache_manager`, and that variable is `protected`. I've had to deprecate it.

To test this, I've used this code snippet. 


```php
class foo extends WCS_Customer_Store_Cached_CPT {

    public function __construct() {
        $bar = $this->post_meta_cache_manager;
    }
}

new foo();

```


```
PHP Notice:  Function foo::post_meta_cache_manager was called <strong>incorrectly</strong>. foo::post_meta_cache_manager has been deprecated, use foo::object_data_cache_manager instead. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 5.2.0.)
```